GravatarApp/Extensions/View+Extensions.swift

```diff
@@ -48,4 +48,12 @@ extension View {
             .padding(.DS.Padding.split)
             .borders(colorScheme: colorScheme)
     }
+
+    func addToastContainer(manager: ToastManager) -> some View {
```

Comment:
This modifier worries me a bit since it wraps the view with a ZStack and you could easily miss that it does that. It's like a hidden bomb.
Can we put the ZStack explicitly to where necessary and remove this modifier?